driver-core/src/main/java/com/datastax/driver/core/querybuilder/BuiltStatement.java

```diff
@@ -293,25 +311,6 @@ public Boolean isIdempotent() {
         return !hasNonIdempotentOps();
     }
 
-    @Override
-    public String toString() {
-        try {
-            if (forceNoValues)
-                return getQueryString();
-            // 1) try first with all values inlined (will not work if some values require custom codecs,
-            // or if the required codecs are registered in a different CodecRegistry instance than the default one)
-            return maybeAddSemicolon(buildQueryString(null, CodecRegistry.DEFAULT_INSTANCE)).toString();
```

Comment:
Just noticed there is a bizarre nuance here.  If we pass in `null` for the first parameter to `buildQueryString` it will inline the value in the query, so if you called `toString()` it would include the parameter in the query string, otherwise if you called `getQueryString()` it wouldn't and rather it would be a value.  This affects something like this (from `Jsr353JsonCodecTest`):

```
insertInto(TABLE1).value("k", "key0").value(quote("miXeD"), "a").value("i", 1).value("f", 1.1).getQueryString()
```

The generated query is: 

```
INSERT INTO test1 (k,"miXeD",i,f) VALUES (?,?,1,1.1);
```

instead of 

```
INSERT INTO test1 (k,"miXeD",i,f) VALUES ('key0','a',1,1.1);
```

However, the BuiltStatement does include the variables when you execute it, so perhaps that doesn't matter so much.